### DISCUSSION TITLE
(100$) Dante (LC) Ticktock Hivemind

### DISCUSSION THREAD

**Discussion by thomas1404-sketch** [Bubberstation/Bubberstation#1782] [ideas/bounties]
I had this idea but im not sure how possible it is to code but it could be a power from a quirk or from the specific head prosthetic (Being Dante's flaming clock head) but it would be like the slime hivemind in which you can link a certain amount of people to your hivemind and they can hear you no matter where, but the extra thing i was thinking was that to anyone unlinked they would hear just TICKTICKTICK coming from the guy. Thus having to force them to use their linked people to be translators if they wished or they could just totally troll them and say things they didnt say but how would they argue against it? Angry Ticktock noises?

For the bounty im offering 100$ on paypal, if i can see it in action working. I did commission someone to do the head prosthetic incase that would be necessary. I am very new to the whole commissioning an entire code thing for ss13. If any questions dm me on discord, my username is thomas14 on it.

Edit: 
![dante](https://github.com/user-attachments/assets/31defff2-6357-4a8d-8948-1dd896a82ce6)

Dante Sprite's were made and are transferred into a DMI file. if needed for the bounty, DM for the sprites

**Comment by Shen37** [comment]


**Reply by thomas1404-sketch** [reply]
Hi, sorry! I thought it was a random scam account, since I’ve had a few trying to friend me, though now I’m beginning to wonder if they’re all actually people just trying to get in contact about this..

**Comment by Snajper2022** [comment]
Is this supposed to be part of an antagonist, or would it be part of a race? Asking to clear up in which way you want it implemented.
